.github/workflows/test-windows.yml

```
@@ -163,25 +163,15 @@ jobs:
   msys:
     runs-on: windows-2019
 
-    strategy:
-      fail-fast: false
-      matrix:
-        mingw: ["MINGW32", "MINGW64"]
-        include:
-          - mingw: "MINGW32"
-            package: "mingw-w64-i686"
-          - mingw: "MINGW64"
-            package: "mingw-w64-x86_64"
-
     defaults:
       run:
         shell: bash.exe --login -eo pipefail "{0}"
     env:
-      MSYSTEM: ${{ matrix.mingw }}
+      MSYSTEM: MINGW64
       CHERE_INVOKING: 1
 
     timeout-minutes: 30
-    name: MSYS2 ${{ matrix.mingw }}
+    name: MSYS2 MINGW64
```

```suggestion
    name: MSYS2 MinGW 64-bit
```

Can use proper name now that it is no longer used in the env block above.